.travis.yml

```diff
@@ -4,15 +4,19 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-26.0.2
-    - android-26
+    - android-27
```

Comment:
You can remove this. It will be installed automatically.

Reply:
varynice.